components/search/search.tsx

```diff
@@ -79,6 +82,30 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
         new SearchUserProvider(actions.autocompleteUsersInTeam),
     ]);
 
+    const isDesktop = isDesktopApp() && isServerVersionGreaterThanOrEqualTo(getDesktopVersion(), '4.7.0');
+    if (enableFindShortcut) {
+        const handleKeyDown = useCallback((e: KeyboardEvent) => {
```

Comment:
the `useCallback` can be removed here, when the function definition is moved inside the useEffect hook

Reply:
I didn't think it worked correctly when I ran the callback from `useEffect` without the `useCallback` hook.

Reply:
Yeah, that's because functions called inside a hook should always be defined in that hook or wrapped in a useCallback to memoize it.

I did not take a look under the hood, but that is how it works.